libraries/AP_HAL_ChibiOS/AnalogIn.h

```diff
@@ -19,8 +19,23 @@
 
 #include "AP_HAL_ChibiOS.h"
 
+// available ADC channels for allocation
```

Comment:
does the changes of ADC3 in this reduce the number of DMA channels available to other peripherals on other H7 boards

Reply:
ADC3 continues to use BDMA like before, so no changes are made in DMA config